clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/config/CloudFoundryConfigurationProperties.java

```diff
@@ -75,7 +75,7 @@ public void destroy() {
   }
 
   @Data
-  public class ClientConfig {
+  public static class ClientConfig {
```

Comment:
Yes, it's to avoid tying it to an object instance, from Effective Java (third edition), item 24:

> Each instance of a nonstatic member class is implicitly associated with an enclosing instance of its containing class. Within instance methods of a nonstatic member class, you can invoke methods on the enclosing instance or obtain a reference to the enclosing instance using the qualified this construct. If an instance of a nested class can exist in isolation from an instance of its enclosing class, then the nested class must be a static member class: *it is impossible to create an instance of a nonstatic member class without an enclosing instance.*